modules/op-installing-pipelines-as-code-on-an-openshift-cluster.adoc

```diff
@@ -7,7 +7,7 @@
 = Installing {pac} on an {product-title}
 
 [role="_abstract"]
-{pac} is installed by default when you install the {pipelines-title} Operator. If you are using {pipelines-shortname} 1.7 or later versions, skip the procedure for manual installation of {pac}.
+{pac} is installed in the `openshift-pipelines` namespace, by default, when you install the {pipelines-title} Operator. If you are using {pipelines-shortname} 1.7 or later versions, skip the procedure for manual installation of {pac}.
```

Comment:
there is no choice to install pac in other namespace, can you confirm @sm43

Reply:
The documentation says when you install PAC manually, it gets installed in the  in the pipelines-as-code namespace.
 
This is written: This command installs Pipelines as Code in the pipelines-as-code namespace and creates user roles and the route URL for the Pipelines as Code event listener.

Reply:
yes with installing operator PAC will be installed in `openshift-pipelines` namespace

we cannot specifically install PAC in a different namepace than pipelines,triggers
all openshift pipelines components will be installed in a single namespace
by default `openshift-pipelines` namespace